wknml/__init__.py

```diff
@@ -177,19 +177,32 @@ class NML(NamedTuple):
     volume: Optional[Volume] = None
 
 
-def __parse_bounding_box(nml_parameters, prefix):
-    boundingBox = None
-    bboxName = prefix + "BoundingBox"
-    if nml_parameters.find(bboxName) is not None:
-        boundingBox = (
-            int(nml_parameters.find(bboxName).get("topLeftX")),
-            int(nml_parameters.find(bboxName).get("topLeftY")),
-            int(nml_parameters.find(bboxName).get("topLeftZ")),
-            int(nml_parameters.find(bboxName).get("width")),
-            int(nml_parameters.find(bboxName).get("height")),
-            int(nml_parameters.find(bboxName).get("depth")),
-        )
-    return boundingBox
+def __parse_user_bounding_boxes(nml_parameters: Element):
+    # ToDo support color, id, name, isVisible attributes
```

Comment:
Can you create a follow-up issue (and potentially link that here) for this?

Reply:
Here you go: https://github.com/scalableminds/wknml/issues/46